pyproject.toml

```diff
@@ -1,3 +1,103 @@
 [build-system]
-requires = ["setuptools >= 64"]
+requires = ["setuptools>=61", "setuptools_scm"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "climada"
+version = "6.0.2-dev"
+description = "CLIMADA in Python"
+readme = "README.md"
+requires-python = ">=3.10,<3.13"
+license = { text = "GNU Lesser General Public License v3 (GPLv3)" }
+authors = [
+    { name = "ETH", email = "schmide@ethz.ch" }
+]
+keywords = ["climate", "adaptation"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: GIS",
+    "Topic :: Scientific/Engineering :: Mathematics"
+]
+dependencies = [
+    "bayesian-optimization",
+    "bottleneck",
+    "cartopy",
+    "cfgrib",
+    "contextily",
+    "dask",
+    "deprecation",
+    "geopandas",
+    "h5py",
+    "haversine",
+    "matplotlib",
+    "netcdf4",
+    "numba",
+    "openpyxl",
+    "overpy",
+    "pandas",
+    "pathos",
+    "peewee",
+    "pillow",
+    "pint",
+    "pycountry",
+    "pyproj",
+    "rasterio",
+    "salib",
+    "scikit-learn",
+    "seaborn",
+    "statsmodels",
+    "sparse",
+    "tables",
+    "tabulate",
+    "tqdm",
+    "xarray",
+    "xlrd",
+    "xlsxwriter"
+]
+
+[project.optional-dependencies]
+doc = [
+    "ipython",
+    "myst-nb",
+    "readthedocs-sphinx-ext>=2.2",
+    "sphinx",
+    "sphinx-book-theme",
+    "sphinx-markdown-tables",
+    "sphinx-design",
+    "sphinx-mdinclude"
+]
+test = [
+    "ipython",
+    "mccabe>=0.6",
+    "pylint>=3.0",
+    "pytest",
+    "pytest-cov",
+    "pytest-subtests"
+]
+dev = [
+    "ipython",
+    "myst-nb",
+    "readthedocs-sphinx-ext>=2.2",
+    "sphinx",
+    "sphinx-book-theme",
+    "sphinx-markdown-tables",
+    "sphinx-design",
+    "sphinx-mdinclude",
+    "mccabe>=0.6",
+    "pylint>=3.0",
+    "pytest",
+    "pytest-cov",
+    "pytest-subtests",
+    "pre-commit"
+]
```

Comment:
This is very inconvenient. I think we should omit the "dev" shortcut then instead of duplicating the lists.